File: 20-sortiranje/redjanje-mehurom.py
def redjajMehurom(niz):
    menjano = True
    while menjano:
        menjano = False
        for x in range(len(niz) - 1):
            if niz[x] > niz[x+1]:
                niz[x], niz[x+1] = niz[x+1], niz[x]
                menjano = True
    return niz

print(redjajMehurom([54,26,93,17,77,31,44,55,20]))

File: 20-sortiranje/redjanje-mehurom.js
const slucajniNiz = require('../utils/slucajniNiz')

function redjajMehurom(niz) {
  let nesredjen = true
  while (nesredjen) {
    nesredjen = false
    for (let x = 0; x < niz.length - 1; x++) {
      if (niz[x] > niz[x+1]) {
        [niz[x], niz[x+1]] = [niz[x+1], niz[x]]
        nesredjen = true
      }
    }
  }
  return niz
}

const niz = slucajniNiz(20)
console.log(niz)

const poredjanNiz = redjajMehurom(niz)
console.log(poredjanNiz)
